spsdk/image/hab/commands/commands.py

```diff
@@ -507,35 +507,40 @@ def __init__(self, cmd: CmdUnlockAbstract) -> None:
         super().__init__(cmd)
 
     @classmethod
-    def load_from_config(cls, config: HabConfig, search_paths: Optional[list[str]] = None) -> Self:
+    def load_from_config(cls, config: HabConfig, search_paths: Optional[list[str]] = None) -> list[Self]:
```

Comment:
In order to parse multiple commands of a same type and keep the order of processed commands, we could perhaps update the API of SecCommandBase(and all its child classes) to something like this:
`load_from_config(cls, command_config: CommandConfig, hab_config: HabConfig, search_paths: Optional[list[str]] = None) -> Self`

Once this is updated in HabContainer class, we will be able to load multiple commands of the same type keeping the order taken from configuration file.